Row.c

```diff
@@ -428,22 +428,22 @@ void Row_printNanoseconds(RichString* str, unsigned long long totalNanoseconds,
    }
 
    unsigned long long totalSeconds = totalMicroseconds / 1000000;
-   unsigned long microseconds = totalMicroseconds % 1000000;
+   unsigned int microseconds = totalMicroseconds % 1000000;
```

Comment:
When I wrote this piece of `Row_printNanoseconds` code, I _did not_ assume the `unsigned int` type would have 32 bits for you (for stricter standard compliance). I know what you are doing, but I would like `uint32_t` be used here instead of `unsigned int`.

Reply:
Also in favour of explicit bit width spec.